src/Reflection/ReflectionParameter.php

```diff
@@ -303,37 +296,6 @@ public function allowsNull(): bool
         return $type->allowsNull();
     }
 
-    /**
-     * Get the DocBlock type hints as an array of strings.
-     *
-     * @return list<string>
-     */
-    public function getDocBlockTypeStrings(): array
```

Comment:
TIL: I use this in https://github.com/Roave/BackwardCompatibilityCheck/blob/11c7b6ff055c00cf9cebcd3f36995261de40385e/src/DetectChanges/BCBreak/PropertyBased/PropertyDocumentedTypeChanged.php#L42 :D 

I'll have to do some legwork to move on from there, I suppose...

Reply:
(good for figuring out what the correct alternatives are)

Reply:
I removed this entire feature from `roave/backward-compatibility-check`